### PR TITLE
fix: Configuring the number of workspaces that a user can create

### PR DESCRIPTION
**Pull request by tstastna**
> Read our [Contribution guide](https://github.com/eclipse/che-docs/blob/master/CONTRIBUTING.adoc) before submitting a PR.

### What does this PR do?
Documents configuring the number of workspaces a user can create. Rewrites configuring the number of workspaces a user can run for consistency.

### Specify the version of the product this PR applies to.
7.31

### PR Checklist

As the author of this Pull Request I made sure that:

- [x] `vale` has been run successfully against the PR branch
- [x] Link checker has been run successfully against the PR branch
- [ ] Documentation describes a scenario that is already covered by QE tests, otherwise an issue has been created and acknowledged by Che QE team
- [ ] Changed article references are updated where they are used (or a redirect has been configured on the docs side):
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/src/services/bootstrap/branding.constant.ts)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)

### CODE REVIEW

**Review by MichalMaler — CHANGES_REQUESTED** [review]
Toto je pouze cvičný "request" :)

**Review by MichalMaler — APPROVED** [review]

**Comment by themr0c** [comment]
Commit by Michal Maléř (48474054+MichalMaler@****.noreply) is causing ECA infringement :/.

**Comment by MichalMaler** [comment]
Hello @tstastna . Please, are we don in here?

**Comment by tstastna** [comment]
Waiting for a peer review from Eva.

File: modules/installation-guide/examples/che-pvc-strategies.adoc
Comment (Eva-Lotte) [inline modules/installation-guide/examples/che-pvc-strategies.adoc:13]: Leaving out "that" or not is not a grammatical but a stylistic question as far as I was able to research this topic online. I didn't find anything in the IBM style guide concerning it. For me it sounds smoother without "that", also it makes the titles shorter. But since it's a stylistic question it's most likely up to you :)

File: modules/installation-guide/partials/con_persistent-volume-configuration.adoc
Comment (Eva-Lotte) [inline modules/installation-guide/partials/con_persistent-volume-configuration.adoc:40]: according to the IBM Style Guide p.179 you shouldn't "continue the lead-in sentence after the list, and do not use a conjunction at the beginning or end of a list item."

File: modules/installation-guide/partials/con_persistent-volume-configuration.adoc
Comment (Eva-Lotte) [inline modules/installation-guide/partials/con_persistent-volume-configuration.adoc:41]: I'd use active voice here "You must reconfigure the workspace PVC access mode ..."
maybe "so that" for clarity?
Also I am pretty sure you don't need a comma here (https://www.grammarly.com/blog/comma-before-so/).

Comment (Eva-Lotte) [inline modules/installation-guide/partials/con_persistent-volume-configuration.adoc:41]: I suppose at this point in the guides it is clear what PVC stands for?

File: modules/installation-guide/partials/con_persistent-volume-configuration.adoc
Comment (Eva-Lotte) [inline modules/installation-guide/partials/con_persistent-volume-configuration.adoc:45]: I suggest "you can use the `common` PVC strategy for clusters with ..." for a more user-centric approach.

File: modules/installation-guide/partials/proc_configuring-namespace-strategies.adoc
Comment (Eva-Lotte) [inline modules/installation-guide/partials/proc_configuring-namespace-strategies.adoc:47]: If you are using "can be running" here, I'd also use it in the file before, where you are using "may". The IBM style guide p.288 generally recommends "can" (can verb. Use to indicate ability. Use instead of "could". To indicate possibility, use "might". To indicate permission, use "may". In most cases, use "can" or "might", not "may".)
Or is this about permission? Then I'd use "may" in both instances.

Comment (Eva-Lotte) [reply modules/installation-guide/partials/proc_configuring-namespace-strategies.adoc:47]: In the rich diff I am able to see this "1. - {prod-short} workspace {orch-namespace} configuration" and there should be a full-stop behind the sentence. Also if there is only one point, consider a bullet point.

Comment (Eva-Lotte) [reply modules/installation-guide/partials/proc_configuring-namespace-strategies.adoc:47]: I am not planning on reading the whole document, but since the rich diff was displaying it, I'll let you know my opinion on this as well ^^":
"The strategy isolates each user in their own {orch-namespace}.
To use the strategy, ..."
Which strategy? I think an adjective would help a lot. The "xy" strategy, or describe what the strategy is for? "The strategy to do xy"

File: modules/installation-guide/partials/proc_using-configmap-to-configure-the-number-of-workspaces-that-a-user-can-create.adoc
Comment (Eva-Lotte) [inline modules/installation-guide/partials/proc_using-configmap-to-configure-the-number-of-workspaces-that-a-user-can-create.adoc]: The first bullet point is rather long and I had to read it twice to get it. Maybe "you installed xy" or "you have xy installed" could help with understandability. But the decision is up to you.

File: modules/installation-guide/partials/proc_using-configmap-to-configure-the-number-of-workspaces-that-a-user-can-create.adoc
Comment (Eva-Lotte) [inline modules/installation-guide/partials/proc_using-configmap-to-configure-the-number-of-workspaces-that-a-user-can-create.adoc]: - delete "the" before user. It's not needed and not consistent.
- here you are not using "that a user can create". If it's a style choice, make it consistent.
- "If the value is a positive integer, users can create that many workspaces", maybe "users can create as many workspaces as the integer number".

File: modules/installation-guide/partials/proc_using-configmap-to-configure-the-number-of-workspaces-that-a-user-can-create.adoc
Comment (Eva-Lotte) [inline modules/installation-guide/partials/proc_using-configmap-to-configure-the-number-of-workspaces-that-a-user-can-create.adoc]: ", by running the following command" (for example). I know this type of procedure writing where you just end with a colon is everywhere, and people like it because it's short and straightforward. However, we should tell the user what to do and use action-oriented verbs for that.

File: modules/installation-guide/partials/proc_using-configmap-to-configure-the-number-of-workspaces-that-a-user-can-create.adoc
Comment (Eva-Lotte) [inline modules/installation-guide/partials/proc_using-configmap-to-configure-the-number-of-workspaces-that-a-user-can-create.adoc]: It's great that you have an introduction sentence! However, this one is telling the user exactly the same thing as the headline. Maybe it would be possible to explain a little bit more what the chapter is about?

File: modules/installation-guide/partials/proc_using-configmap-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc
Comment (Eva-Lotte) [inline modules/installation-guide/partials/proc_using-configmap-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc]: See other comments :)

File: modules/installation-guide/partials/proc_using-configmap-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc
Comment (Eva-Lotte) [inline modules/installation-guide/partials/proc_using-configmap-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc]: I would put this information in a note. It's too explanatory for a prerequisite. I suppose the two criteria will need to be listed in the prerequisites nonetheless.

File: modules/installation-guide/partials/proc_using-configmap-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc
Comment (Eva-Lotte) [inline modules/installation-guide/partials/proc_using-configmap-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc]: by running the following command

Comment (Eva-Lotte) [reply modules/installation-guide/partials/proc_using-configmap-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc]: since this is a procedure and not general information, would "configure your number of workspaces" make sense? Or is this some sysadmin thing, where you determine what the users can and can't do (obviously clueless here :D)

File: modules/installation-guide/partials/proc_using-the-operator-to-configure-the-number-of-workspaces-that-a-user-can-create.adoc
Comment (Eva-Lotte) [inline modules/installation-guide/partials/proc_using-the-operator-to-configure-the-number-of-workspaces-that-a-user-can-create.adoc]: Introductory sentence is just repeating the headline.

File: modules/installation-guide/partials/proc_using-the-operator-to-configure-the-number-of-workspaces-that-a-user-can-create.adoc
Comment (Eva-Lotte) [inline modules/installation-guide/partials/proc_using-the-operator-to-configure-the-number-of-workspaces-that-a-user-can-create.adoc]: see other comments.

File: modules/installation-guide/partials/proc_using-the-operator-to-configure-the-number-of-workspaces-that-a-user-can-create.adoc
Comment (Eva-Lotte) [inline modules/installation-guide/partials/proc_using-the-operator-to-configure-the-number-of-workspaces-that-a-user-can-create.adoc]: by doing xy.

File: modules/installation-guide/partials/proc_using-the-operator-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc
Comment (Eva-Lotte) [inline modules/installation-guide/partials/proc_using-the-operator-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc]: see other comments

File: modules/installation-guide/partials/proc_using-configmap-to-configure-the-number-of-workspaces-that-a-user-can-create.adoc
Comment (Eva-Lotte) [reply modules/installation-guide/partials/proc_using-configmap-to-configure-the-number-of-workspaces-that-a-user-can-create.adoc]: It is a bit strange that you are mentioning "the following placeholder" only to insert this table. How about "You have determined the value of the `_<number-of-workspaces>_` placeholder. And then add a note: 

[NOTE]
====
If the  value is -1 ...
====

At this point Oss would send this article about tables to read, so I'll follow her example here :) :
https://medium.com/@heyoka/lets-bring-table-to-the-table-again-f1ae751159d5

File: modules/installation-guide/partials/proc_using-the-operator-to-configure-the-number-of-workspaces-that-a-user-can-create.adoc
Comment (MichalMaler) [reply modules/installation-guide/partials/proc_using-the-operator-to-configure-the-number-of-workspaces-that-a-user-can-create.adoc]: @tstastna Hello Tereza. Try to ask the dev you work on this with about how this setup could enhance user experience and add it there :)

File: modules/installation-guide/examples/che-pvc-strategies.adoc
Comment (tstastna) [reply modules/installation-guide/examples/che-pvc-strategies.adoc:13]: Naomi Deevy recommended adding 'that' in this instance for easier translation.

Comment (max-cx) [reply modules/installation-guide/examples/che-pvc-strategies.adoc:13]: That makes sense for global audiences.

File: modules/installation-guide/partials/proc_using-configmap-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc
Comment (max-cx) [reply modules/installation-guide/partials/proc_using-configmap-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc]: The procedure sentence could instead provide more details as to what action exactly (to type/save) or where exactly (in which file) the user is supposed to perform, so as not to repeat the same phrase from the heading and intro par.

File: modules/installation-guide/partials/proc_using-the-operator-to-configure-the-number-of-workspaces-that-a-user-can-create.adoc
Comment (max-cx) [reply modules/installation-guide/partials/proc_using-the-operator-to-configure-the-number-of-workspaces-that-a-user-can-create.adoc]: "This procedure describes how to ..." might be classified as self-referential text according to ibm_style.
I haven't read anything outside of the diff view, but this sentence could also be reworded into a general statement of fact like so:
"ConfigMap enables you/admin/users to configure the number of workspaces that a user (any user? you/reader?) can create."

File: modules/installation-guide/partials/assembly_configuring-the-number-of-workspaces-that-a-user-can-create.adoc
Comment (max-cx) [inline modules/installation-guide/partials/assembly_configuring-the-number-of-workspaces-that-a-user-can-create.adoc:8]: Self-referential text according to ibm-style.

File: modules/installation-guide/partials/proc_configuring-namespace-strategies.adoc
Comment (max-cx) [reply modules/installation-guide/partials/proc_configuring-namespace-strategies.adoc:47]: Just looking at this sentence without any context:
"Only one workspace can be running" because the reader can allow it to run or because the reader is the one who makes it run?

File: modules/installation-guide/partials/proc_using-configmap-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc
Comment (max-cx) [inline modules/installation-guide/partials/proc_using-configmap-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc]: "Therefore to increase the number of simultaneously running workspaces, the combination of PVC strategy and access mode must meet the following criteria" - consider rewording this sentence without removing any details, as it's difficult to follow what it means.

**Review by max-cx — CHANGES_REQUESTED** [review]
'Requesting changes' here only because I'm not allowed to submit general feedback.

File: modules/installation-guide/partials/proc_using-configmap-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc
Comment (tstastna) [reply modules/installation-guide/partials/proc_using-configmap-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc]: This is from an outdated version. In the current version, the sentence is more concise.

File: modules/installation-guide/partials/proc_configuring-namespace-strategies.adoc
Comment (tstastna) [reply modules/installation-guide/partials/proc_configuring-namespace-strategies.adoc:47]: I believe the user can only run one workspace because that's the default configuration. In this file, I only updated the reference. I did not attempt to rewrite the content.

**Comment by tstastna** [comment]
Hello @tolusha , can you please review?

File: modules/installation-guide/partials/proc_configuring-namespace-strategies.adoc
Comment (nickboldt) [reply modules/installation-guide/partials/proc_configuring-namespace-strategies.adoc:47]: "By default, only one workspace in the same {orch-namespace} can be running at one time."
could also be:
"By default, users are only allowed to run a single workspace per {orch-namespace}, but this limit can be increased by an administrator to allow multiple concurrent workspaces to run in parallel, or to run unlimited workspaces."

See https://github.com/eclipse-che/che-server/blob/main/assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties where `che.limits.user.workspaces.run.count=1` is mentioned.

**Review by nickboldt — COMMENTED** [review]
17 files changed just for some filename refactoring? Oof. Painful but +1.

Also there's a lot of individual files just to explain the two ways to override a che property with a custom resource or helm chart... but hey. 

* che.limits.user.workspaces.run.count, or
* che.limits.userWorkspacesRunCount, or
* CHE_LIMITS_USER_WORKSPACES_RUN_COUNT ... it's all the same thing, under the covers. :)

**Comment by themr0c** [comment]
@nickboldt Totally correct, the concept is always the same, but the procedure differs.

**Review by AndrienkoAleksandr — APPROVED** [review]

File: modules/installation-guide/partials/proc_using-the-operator-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc
Comment (mmorhun) [inline modules/installation-guide/partials/proc_using-the-operator-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc]: `how to use the Operator` -> `how to configure ${prod-short}` (as Che Operator doesn't deal with user workspaces directly).

File: modules/installation-guide/partials/proc_using-the-operator-to-configure-the-number-of-workspaces-that-a-user-can-create.adoc
Comment (mmorhun) [inline modules/installation-guide/partials/proc_using-the-operator-to-configure-the-number-of-workspaces-that-a-user-can-create.adoc]: The same as for run count

File: modules/installation-guide/partials/proc_using-configmap-to-configure-the-number-of-workspaces-that-a-user-can-create.adoc
Comment (mmorhun) [inline modules/installation-guide/partials/proc_using-configmap-to-configure-the-number-of-workspaces-that-a-user-can-create.adoc]: To me it doesn't make sense. You cannot use a ConfigMap to install something. ConfigMap is a set of keys/values, i.e. data.
Admin can change Che settings by editing fields of the configmap but only in case if Helm installer was used to deploy Che. There is no mention of it here.

File: modules/installation-guide/partials/proc_using-configmap-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc
Comment (mmorhun) [inline modules/installation-guide/partials/proc_using-configmap-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc]: The same as above

File: modules/installation-guide/partials/proc_using-the-operator-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc
Comment (tstastna) [reply modules/installation-guide/partials/proc_using-the-operator-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc]: Thank you for the clarification.

**Review by mmorhun — APPROVED** [review]

**Review by MichalMaler — APPROVED** [review]

**Pull request merged**
